### PR TITLE
Adds a "Code Grey" security level

### PR DESCRIPTION
**Pull request by Ty-Omaha**
This is to combat the greytiders. 
Code Grey can be activated via the admin "secrets" tab which does pretty much the same as Gamma in terms of physical changes but doesn't unlock the armory and is specifically for greytides, allowing sec to go ham on Greytider McGrey and his buds.

This was just sort of me messing around with the code because lately I've been talking about "code grey" but this was some good practice for me nonetheless. 

7f3e60d3254a01a39f946f6c150c0c8c

848550abd75777316b583dcf1154045b

9322d1aa06f189275b190a7a0d3fecaa

:cl:
:rscadd: Adds in "Code Grey"
:rscadd: Adds in Fire Alarm and Status Display sprites for "Code Grey"
/:cl:

### CODE REVIEW

**Comment by Wheatley29** [comment]
Yes

**Comment by FlattestGuitar** [comment]
I can't tell if this is a meme or not.

:+1:

**Comment by Ty-Omaha** [comment]
It /was/ a meme, but now it's not. >:D

**Comment by Wheatley29** [comment]
Clearly serious, but we need to add this

**Comment by theColdflame** [comment]
:-1:

this just glorifies shittery 

if nonantags are causing enough trouble that admins have to step in, it's beyond the point of ICE resolution

**Comment by Wheatley29** [comment]
If people consider glory being harmbatoned then that's an issue itself

**Comment by chopchop1614** [comment]
No, sorry but no. Its really not needed, also why would you create an alert level that just allows security to "go ham" on the greytides? Try helping them or something.

File: code/modules/security_levels/security levels.dm [hunk 1]
```diff
@@ -146,6 +162,8 @@
 			return "gamma"
 		if(SEC_LEVEL_EPSILON)
 			return "epsilon"
+		if (SEC_LEVEL_GREY)
```
Comment (FlattestGuitar) [inline code/modules/security_levels/security levels.dm]: Only YOU can prevent whitespace abuse.

**Comment by Necaladun** [comment]
I love this and would love to spawn shitloads of sNPC greytiders then turn it on.

**Comment by MindLeaker** [comment]
I have a lot of doubts here:
- If we give Security a valadlevel where they just fucking murderbone an entire sector of crew because an admin pressed a button, what does that say about the server's RP level as a whole? This is essentially a two-department highlander

- Newer players trying to learn the ropes of the game as a Civilian will be caught up by this an subsequently killed for being a member of the Civilian staff, since this doesn't draw a line between good and bad civvies.
 

- There are much better "round-enders" already in place that make it fun for the whole station, not just giving Security a free DS pass because "Why not?"

- If this is a prank then I'm an idiot and deserve to get tossed in perma for a few days

**Comment by MrFroztee** [comment]
The terminology of "Code Grey" just doesn't give off a severe tone, considering the severity that can come with Greytide.

But in the use of such terms, it makes the alert level sound like a joke. Besides, Security is already equipped and ready to take on greyshits, and should be doing so regardless.

**Comment by Belario** [comment]
YES

The tide is a legitimate problem and needs a legitimate response.
Just toss in some "rules of engagement" and we're golden.

**Comment by Necaladun** [comment]
As I see it, this is the equivalent of highlander mode in seriousness.

**Comment by MindLeaker** [comment]
This also has the equivalent of Mutiny in engagement, unlike in Highlander where the entire crew is involved, this is two quarters of the station duking it out, while Medbay, Cargo, Science, and Engineering just... watch? It seems redundant and abusable honestly.

**Comment by Dinarzad** [comment]
If it's an admin only alert level (Similar to Gamma)
Then what does the "Silliness" of it matter? There's no physical way the Crew can call it, and if the round has become such an absolute shitshow due to Greytide that an admin calls the alert, a chuckle or two might be kinda needed;

**Comment by Lady-Luck** [comment]
I too hate fun adminbus things.

**Comment by Wheatley29** [comment]
Fox McCloud hates fun confirmed

**Comment by Ty-Omaha** [comment]
!!FUN!!

**Comment by Spacemanspark** [comment]
If it's admin only, then sure, why not.

File: code/modules/security_levels/security levels.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/security_levels/security levels.dm]: Nah it's fine.

**Comment by Earthdivine** [comment]
If it's admin only, sure.

But this command also is basically saying "Free valids on the newcomers!" I do not like that as that will outright ruin a new players introduction.

**Comment by Taaketa** [comment]
It's dumb. Why highlight a meme, even if it's admin only? Plus all you're gonna get as an admin team is centcomm faxes asking for greytide alert.

**Comment by IcyV** [comment]
>But this command also is basically saying "Free valids on the newcomers!" I do not like that as that will outright ruin a new players introduction.

That's my concern with it. I've noticed a lot of people just generally throw new players into the category of greytide here recently. Not exactly easy for them to get into it when they're being ignored, hunted, or talked down to. I'm just worried on how it will be used.

**Comment by Kyep** [comment]
I like the concept of having more special security levels.

I don't like this implementation, because:
- Every other special level (gamma, delta, epsilon, etc) is a greek letter. This is the only one that isn't.
- I just can't shake the association with "condition grey" from Star Trek, and "beige alert" from Futurama. "Gray Alert" just isn't... punchy.
- I worry that actually activating this gives both greytide and sec a license to kill each other, which might be exactly what the greytide want, and thus encourage them to greytide even more in future.

It would be good to have an alert level which says "everything is going to hell" but doesn't unlock gamma armory. It should have a greek-letter name, though. Like... "Sigma". Something to use when the situation is dire but the Gamma armory won't help matters. The CC announcement for it shouldn't reference orders - that's what a separate announcement is for.

File: code/modules/security_levels/security levels.dm (hunk 1 above)
Comment (taukausanake) [inline code/modules/security_levels/security levels.dm]: Pretty sure @tigercat2000 is going to bite someone's head off

Comment (tigercat2000) [inline code/modules/security_levels/security levels.dm]: :hocho:

Comment (Ty-Omaha) [inline code/modules/security_levels/security levels.dm]: wtf are you guys on about theres no extra line here????????????????????????????????????????????????
line 166 is clearly: return"grey" 
>:(
line 203 though is a different story...

**Comment by Vivalas** [comment]
I agree that this is bad, but if sec were indeed able to kill greytiders the problem would mostly be gone.

**Comment by Ty-Omaha** [comment]
@Vivalas But what about all the !!FUN!!

**Comment by Plotron** [comment]
I don't even...

**Comment by Fox-McCloud** [comment]
The memessss

**Comment by Crazylemon64** [comment]
"Phi Omicron Kappa" alert, where the station turns into a frat house

**Comment by Ty-Omaha** [comment]
@Fox-McCloud fox noooooooooooooooooooooooooooooooo you hit the wrong button you were supposed to hit merge, not close!!!!! :(

**Comment by Lady-Luck** [comment]
Mmmn yeah I think we actually need this

**Pull request closed without merging**